tests/resources/sample_config.yaml

```
@@ -23,6 +23,8 @@ grains:
   threshold_std_dev: 1.0
   threshold_absolute_lower: -1.0
```

We could correct this too...

```suggestion
  threshold_absolute_lower: 1.0
```